editor/import/scene_import_settings.cpp

```diff
@@ -411,6 +412,11 @@ void SceneImportSettings::_fill_scene(Node *p_node, TreeItem *p_parent_item) {
 			contents_aabb.merge_with(aabb);
 		}
 	}
+
+	Skeleton3D *skeleton = Object::cast_to<Skeleton3D>(p_node);
+	if (skeleton) {
+		bones_mesh_preview->set_mesh(Skeleton3DGizmoPlugin::get_bones_mesh(skeleton, -1, true));
+	}
```

Comment:
This relationship doesn't seem right to me. We shouldn't rely directly on plugin code here, that defeats the purpose of it being a plugin. We also can't use the same pattern going forward if we need to bring more viewport gizmos to the advanced import dialog preview.

I think it's time to integrate viewport/gizmo plugins into this preview, or alternatively devise a separate pluggable interface.

Reply:
We talked about this at the animation asset meeting and we would like this change. Saracen has already approved the technical aspects.

I don't think this is such a big issue since it's a static function and both are in editor/ so there is no linking problem.
@AThousandShips would it be possible to review this usage of a plugin function from the import code and determine if this is ok?

Reply:
I'll take a look :)

Reply:
I don't think there's any issue with this, both are fully contained in the editor, and it saves a lot of code duplication, could theoretically move it to Skeleton3D or similar but it being a plugin here doesn't change anything in my opinion, as it's not a module or similar but fully integrated